requirements.txt

```diff
@@ -19,3 +19,4 @@ pytest-ordering
 sphinx-jsonschema
 pytest-console-scripts
 pre-commit==2.10.1
+SimpleITK
```

Comment:
To be discussed in the upcoming IvadoMed meetings to ensure we are okay adding the additional dependencies..